app/views/general_shopping_lists/general.html.erb

```diff
@@ -0,0 +1,28 @@
+<section class="container flex food-placeholder">
+<div class="food flex">
+  <h3 class="title">General Shopping List</h3>
+  <div class="sum flex">
+    <p>Amount of food to buy: <%= @general_shopping_lists.count %> </p>
+    <p>Total value of food needed: $<%= @general_shopping_lists.map { |food| food.price}.sum %></p>
+  </div>
+
+  <table>
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>Quantity</th>
+        <th>Price</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @general_shopping_lists.each do |food| %>
+        <tr>
+          <td><%= food.name %></td>
+          <td><%= food.quantity %> <%= food.measurement_unit %></td>
+          <td>$<%= food.price %></td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+</div>
+</section>
```

Comment:
- [ ] Kindly also make sure to count the total food items and total price of the missing food as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=of%20that%20user\).-,Should%20count%20the%20total%20food%20items%20and%20total%20price%20of%20the%20missing%20food.,-Make%20sure%20there).